services/wfm/src/main/java/org/openkilda/wfm/topology/flow/bolts/CrudBolt.java

```diff
@@ -795,5 +802,14 @@ public void sendFlowCommands(@NonNull String flowId, @NonNull List<CommandGroup>
                     System.currentTimeMillis(), correlationId);
             outputCollector.emit(stream.toString(), tuple, new Values(message, flowId));
         }
+
+        @Override
+        public void sendPeriodicPingNotification(@NonNull String flowId, boolean enabled) {
+            CommandMessage message = new CommandMessage(
+                    new PeriodicPingCommand(flowId, enabled),
+                    System.currentTimeMillis(), correlationId);
+
+            outputCollector.emit(stream.toString(), tuple, new Values(correlationId, message));
```

Comment:
should it be `StreamType.PING.toString()` instead of `stream.toString()` ?

Reply:
No, it should be fine, p.s. this aligned with `sendFlowCommands`